src/call.cpp

```diff
@@ -941,6 +941,38 @@ bool call::connect_socket_if_needed()
 
         sipp_customize_socket(call_socket);
 
+        if (peripsocket) {
+            struct sockaddr_storage saddr;
+            char peripaddr[256];
```

Comment:
Shouldn't this be `SIPP_MAX_MSG_SIZE`? Or am I mistaken? To my understanding, it looks like `getFieldFromInputFile` assumes that buffer length.